### ISSUE TITLE
Unable to run visualize.py because URL is not accessible

### ISSUE THREAD

**Issue by lhs4297** [mit-han-lab/spvnas#115]
Hello, Thanks for your work. SPVNAS is indeed a great tool.

I'm trying to run visualize.py , but there's a problem and it's not working.

In this section, minkunet, spvcnn, spvnas_specialized from moel_zoo.py, for example, there is code to download and import weights from url such as 'https://hanlab.mit.edu/files/SPVNAS/spvnas_specialized/' '

However, I cannot access this url, and it says that I cannot find the page even if I search directly.

In conclusion, visualize.py doesn't work. How do I solve this problem?

**Comment by zhijian-liu** [comment]
Please use `hanlab18.mit.edu` in place of `hanlab.mit.edu` in the link as a temporary solution.
